osu.Game/Configuration/SessionStatics.cs

```diff
@@ -20,6 +20,8 @@ protected override void InitialiseDefaults()
             SetDefault(Static.MutedAudioNotificationShownOnce, false);
             SetDefault(Static.LowBatteryNotificationShownOnce, false);
             SetDefault(Static.LastHoverSoundPlaybackTime, (double?)null);
+            SetDefault(Static.ReplayPlaybackSettingExpanded, true);
```

Comment:
I honestly think this should start closed by default. I also think the controls should automatically expand similar to how the editor ones do, which may mean we can also avoid adding this to session statics.

Reply:
having to wait for the controls to expand every time you want to change something sounds like a pain (i.e. i don't like how it's done in the editor)

![](https://wala.s-ul.eu/V5jbPGVd.gif)

![](https://wala.s-ul.eu/7jMYvGMe.gif)

something like what this pr is doing would be way more comfortable.

Reply:
For what it's worth, the menus could be made to expand on hover without a delay as well

Reply:
without polluting this branch and reverting the changes to `SessionStatics`, is the below remotely correct?
https://github.com/integerrr/osu/commit/e8c863340b97059cf7c4181796ef97474d5de2c9